### ISSUE TITLE
npm install does not install bower dependencies 

### ISSUE THREAD

**Issue by jstncno** [linnovate/mean#851]
I am following the instructions on the [Wiki](https://github.com/linnovate/mean/wiki/Getting-Started).

When I open the page on localhost:3000, a blank page appears and and the app crashes:

```
$ grunt --force
Running "clean:0" (clean) task

Running "jshint:all" (jshint) task

   packages/system/app.js
     10 |var System = new Module('system');
             ^ Redefinition of 'System'.

>> 1 error in 47 files
Warning: Task "jshint:all" failed. Used --force, continuing.

Running "csslint:src" (csslint) task
>> 5 files lint free.

Running "concurrent:tasks" (concurrent) task
Running "nodemon:dev" (nodemon) task
Running "watch" task
Waiting...
[nodemon] v1.0.20
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node --debug server.js`
debugger listening on port 5858
Mean app started on port 3000 (development)
GET / 200 31.274 ms - -

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: ENOENT, open '/Users/justincano/Projects/mean/testApp/bower_components/bootstrap/dist/css/bootstrap.css'
[nodemon] app crashed - waiting for file changes before starting...
```

I guess it turns out that it can't locate the bootstrap files, so I need to run a `bower install` myself. According to the docs (and the introductory [YouTube video](https://www.youtube.com/watch?v=3gOZCDuQ6xY)), `npm install` should also install bower dependencies.

EDIT:
After playing around a bit, I found out that running `bower install` will only install the dependencies if the app was installed using `sudo npm install`. If you run `npm install` without sudo, `bower install` won't do anything.
